webrender/res/ps_border_corner.vs.glsl

```diff
@@ -32,6 +42,34 @@ void set_edge_line(vec2 border_width,
     vColorEdgeLine = vec4(outer_corner, vec2(-gradient.y, gradient.x));
 }
 
+void write_color(vec4 color0, vec4 color1, float style, vec2 delta) {
+    vec4 modulate = vec4(1.0 + 0.3 * delta.x,
+                         1.0 - 0.3 * delta.x,
+                         1.0 + 0.3 * delta.y,
+                         1.0 - 0.3 * delta.y);
+
+    switch (int(style)) {
+        case BORDER_STYLE_GROOVE:
+            vColor00 = vec4(color0.rgb * modulate.y, color0.a);
```

Comment:
nit: could be rewritten by turning the match to produce a `vec4 modulate` vector, which is then used to compute `vColorXY`